features/c-mode-basic-operators.feature

```diff
@@ -189,27 +204,26 @@ Feature: C basic operators
     When I type "result = foo * *bar"
     Then I should see "result = foo * *bar"
 
-  # # This doesn't actually test what it should, it always passes. Possible
-  # # ecukes bug?
```

Comment:
This looks indeed like an ecukes bug. Changing (call-interactively #'move-end-of-line) to (funcall #'move-end-of-line nil) fixes this (see electric-spacing-steps.el), however, I currently don't understand why. I created a similar espuds ert test, but there it is working fine. Not sure if the py-string syntax does anything fancy that leads to this. I used plain (insert) in the ert test and didn't look at the py-string implementation.